hw/ip/rv_core_ibex_peri/data/rv_core_ibex_peri.hjson

```diff
@@ -0,0 +1,156 @@
+// Copyright lowRISC contributors.
```

Comment:
Is there a downside of maintaining these sources in `hw/ip/rv_core_ibex` itself?

Reply:
no, we can move it over there if you prefer.